serverless/aws/lambda-tests/src/test/scala/sttp/tapir/serverless/aws/lambda/tests/AwsLambdaSamLocalHttpTest.scala

```diff
@@ -0,0 +1,67 @@
+package sttp.tapir.serverless.aws.lambda.tests
+
+import cats.effect.IO
+import org.scalatest.Assertions
+import org.scalatest.compatible.Assertion
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers._
+import sttp.capabilities.WebSockets
+import sttp.capabilities.fs2.Fs2Streams
+import sttp.client3.{basicRequest, _}
+import sttp.model.{Header, Uri}
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.tests.backendResource
+
+class AwsLambdaSamLocalHttpTest extends AnyFunSuite {
```

Comment:
again, here I would add a comment that this assumes that a sam-local process is started (this setup is non-obvious so needs documenting)